docs/docs/getting_started.md.in

```diff
@@ -177,10 +182,29 @@ Where the entries are defined as (* indicates required field):
 : Controls the verbosity of error messages. Default is *false*
 
 - enable_pre_sleep
-: ? Default is *false*
+: Force ILLIXR to sleep for the given number of seconds before starting the plugins (this is useful for attaching a debugger), Default is 0, indicating no sleeping.
 
 In general, you should not edit a [profile][23] file directly. The exception to this is when you are testing things on your own machine. [Profile][23] files are generated automatically from the master `profiles/plugins.yaml` during the cmake configuration stage. This is done so that any changes to a [profile][23] or the addition or removal of a [plugin][22] can be managed from a single file. The build system will generate an *illixr.yaml* file which contains entries from the command line and any input profile file and can be freely edited (it is generated every time `cmake` is called).
 
+## Environment Variables
+
+ILLIXR and several of its dependencies use environment variables to steer processing and define quasi-constant values. Traditionally, one would use std::getenv and std::setenv to get and set these items. However, in ILLIXR the switchboard acts as an interface to these functions with `get_env` (returns std::string), `get_env_bool` (returns bool), `get_env_char` (returns char*), and `set_env`. Using the switchboard interfaces allows for environment variables to be set from the command line and/or yaml file. It is strongly encouraged to the switchboard interface wherever possible. Any items given on the command line which do not match expected command line arguments are considered to be environment variables. For example:
+
+```bash
+main.dbg.exe --yaml=my.yaml --DATA_SOURCE=/home/my/data --PROCESS_DATA
```

Comment:
This does not work for me, trying with ``GTSAM_INTEGRATOR_LOG_LEVEL=error``. Examining the source code suggests that this should not work as well. 

I think it's fine if this isn't supported. It's not so hard to set the environment variable before the executable, rather than after. We just shouldn't mention it in the documentation then.